bin/export-test-env

```diff
@@ -0,0 +1,6 @@
+#! /bin/bash
```

Comment:
I would provide a more descriptive name than mere `test`. Maybe `multiproposer-test` or something like that